src/sequence/cache/strategies/default.js

```diff
@@ -0,0 +1,46 @@
+const { assign, omit } = require("lodash");
+
+const { sequence: makeNewSequence } = require("../../sequence");
+const { SequenceCache } = require("../sequence-cache");
+
+
+const cache = Object.create(null);
```

Comment:
Again, maybe consider instantiable cache objects for renderers instead of global mutable singletons.